tests/requirements.txt

```diff
@@ -1,3 +1,3 @@
-pytest===6.1.2
-pytest-asyncio==0.14.0
-websockets==8.1
+pytest==7.2.0
```

Comment:
Should we update examples/requirements.txt as well?

Reply:
updated